### PR TITLE
Add Wemos Lolin32-lite variant

### PR DESCRIPTION
**Pull request by henricazottes**
Same pins file except that the builtin led is on gpio 22 instead of gpio 5.

### CODE REVIEW

**Comment by me-no-dev** [comment]
sure, but there is no board definition that uses this variant?

**Comment by henricazottes** [comment]
I admit I didn't check.
But even if there is one compatible, wouldn't a user look for the variant corresponding to the board he is using ?

For my case, I tried the Lolin 32 variant, it didn't work, I checked what could be the problem and found it was just the LED pin definition. I kinda know what I'm doing, maybe less skilled users would just have conclude that their board is dead :/

**Comment by me-no-dev** [comment]
Without a board definition that uses variant ```lolin32-lite``` this file is never used ;)
Either add a board definition or close this :)

**Comment by me-no-dev** [comment]
..... ```lolin32``` already exists! maybe you meant to name it ```lolin32-lite```? Please pay attention!

**Comment by henricazottes** [comment]
Thanks for the tip :)

**Comment by henricazottes** [comment]
Anything else to correct/change ?

**Comment by me-no-dev** [comment]
you are running an old code base ;) boot=bootloader is no longer correct! Please update your fork and change this to boot=dio!!!!

you are running an old code base ;) boot=bootloader is no longer correct! Please update your fork and change this to boot=dio!!!!

**Comment by henricazottes** [comment]
Done :)

**Comment by henricazottes** [comment]
@me-no-dev anything else ?

File: boards.txt [hunk 1]
```diff
@@ -335,6 +335,49 @@ lolin32.menu.UploadSpeed.512000.upload.speed=512000
...
+lolin32-lite.upload.maximum_size=1310720
+lolin32-lite.upload.maximum_data_size=294912
+lolin32-lite.upload.wait_for_upload_port=true
+
+lolin32-lite.serial.disableDTR=true
+lolin32-lite.serial.disableRTS=true
+
+lolin32-lite.build.mcu=esp32
+lolin32-lite.build.core=esp32
+lolin32-lite.build.variant=lolin32-lite-lite
```
Comment (wunderlins) [inline boards.txt]: This is wrong, the variant folder is `lolin32-lite` in this patch. Therefore the correct value would be:
`lolin32-lite.build.variant=lolin32-lite`

File: boards.txt [hunk 2]
```diff
@@ -335,6 +335,49 @@ lolin32.menu.UploadSpeed.512000.upload.speed=512000
...
+lolin32-lite.build.board=LOLIN32-LITE
```
Comment (wunderlins) [inline boards.txt]: on my system, with arduino 1.8.5 (linux-x64) it only works if I change this to `lolin32-lite.build.board=LOLIN32`

**Comment by evark** [comment]
It is not just the LED_BUILTIN different.
Pin 21 (default SDA) , does not exist in lolin32-lite. 
I2C pins must relocated from 21/22 to another location, eg 22/23.

File: boards.txt (hunk 1 above)
Comment (rhyslbw) [reply boards.txt]: Correct in #1076

File: boards.txt (hunk 2 above)
Comment (rhyslbw) [reply boards.txt]: Correct in #1076

**Comment by rhyslbw** [comment]
Another PR with the variant here #1076

**Comment by DavidAntliff** [comment]
@me-no-dev why did you close this?

**Pull request closed without merging**